plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx

```diff
@@ -185,6 +187,15 @@ export const NotificationsTable = ({
   const compactColumns = React.useMemo((): TableColumn<Notification>[] => {
     const showToolbar = notifications.length > 0;
 
+    const NotificationIcon = (props: { notification: Notification }) => {
```

Comment:
Can we move definition of this component out of `NotificationsTable`? Up in this file.
The `useApp` will need to be called again in it.

Reply:
extracted notification component to separate file.